packages/edit-site/src/components/page-patterns/style.scss

```diff
@@ -11,11 +11,6 @@
 	.dataviews-view-table & {
 		width: 96px;
 		flex-grow: 0;
-		border-radius: 2px;
-
-		.page-patterns-preview-field__button {
-			border-radius: 2px;
-		}
```

Comment:
I don't see any reason for the preview thumbnail to have a different radius based on layout selection.